test/unit/gateways/sum_up_test.rb

```diff
@@ -0,0 +1,361 @@
+require 'test_helper'
+
+class SumUpTest < Test::Unit::TestCase
+  def setup
+    @gateway = SumUpGateway.new(
+      token: 'sup_sk_ABC123',
+      pay_to_email: 'example@example.com'
+    )
+    @credit_card = credit_card
+    @amount = 100
+
+    @options = {
+      payment_type: 'card',
+      billing_address: address,
+      description: 'Store Purchase'
+    }
+  end
+
+  def test_successful_purchase
```

Comment:
💬 Comment:

given that response is going to be fixed the only point of having a unit test here is to actually check how the request is being built, so you can confirm that values are in the proper place and format.